### PR TITLE
Feature/form validation

### PR DESCRIPTION
**Pull request by PrinceD96**
Registration rewrite with validation schema using Formik + Yup. There was no validation before. The form does nothing when submitted yet. That would be a separate PR😉

### CODE REVIEW

File: src/components/authentication/Login.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+						fullWidth
+						variant='contained'
+						color='primary'
+						className={classes.submit}
+					>
+						Sign In
+					</Button>
+					<Grid container justify='center'>
+						{/* <Grid item xs>
+							<Link style={{ fontSize: 14, color: '#0000EE' }}>
```
Comment (raudelf) [inline src/components/authentication/Login.js:79]: We need to get rid of commented out code before we continue with this

File: src/components/navbar.js [hunk 2]
```diff
@@ -46,279 +45,491 @@ import SendIcon from '@material-ui/icons/Send';
...
+				<AppBar
+					position='fixed'
+					style={{
+						boxShadow: 'none',
+						backgroundColor: '#0F3088',
+						height: '8vh'
+					}}
+				>
+					<Toolbar style={{ minHeight: '0', color: 'black' }}>
+						{/* <Typography variant="h3" className={classes.title} style={{border: "1px solid black", width: "400px"}}>
```
Comment (raudelf) [inline src/components/navbar.js:152]: commented out code

File: src/components/navbar.js [hunk 3]
```diff
@@ -46,279 +45,491 @@ import SendIcon from '@material-ui/icons/Send';
...
+							style={{
+								display: 'flex',
+								flexDirection: 'row',
+								width: '100%',
+								justifyContent: 'flex-end'
+							}}
+						>
+							<List>
+								<div style={{ width: '75%', color: '#0D47A1' }}>
+									{/* <IconButton
```
Comment (raudelf) [inline src/components/navbar.js:178]: Commented out code

File: src/components/navbar.js [hunk 4]
```diff
@@ -46,279 +45,491 @@ import SendIcon from '@material-ui/icons/Send';
...
+					style={{
+						position: 'fixed',
+						boxShadow: 'none',
+						backgroundColor: '#0F3088',
+						height: '8vh',
+						width: '100%'
+					}}
+				>
+					<Toolbar style={{ minHeight: '0', color: 'black' }}>
+						{/* <Typography variant="h3" className={classes.title} style={{border: "1px solid black", width: "400px"}}>
```
Comment (raudelf) [inline src/components/navbar.js:310]: commented out code

File: src/components/navbar.js [hunk 5]
```diff
@@ -46,279 +45,491 @@ import SendIcon from '@material-ui/icons/Send';
...
+							<div style={{ width: '100%' }}>
+								<List>
+									<div
+										style={{
+											display: 'flex',
+											justifyContent: 'flex-end',
+											alignItems: 'center'
+										}}
+									>
+										{/* <ListItem style={{ width: '15%', minWidth: "100px", fontSize: '1rem', textAlign: 'center' }} button onClick={handleClose} component={Link} to="/">
```
Comment (raudelf) [inline src/components/navbar.js:345]: commented out code

File: src/components/navbar.js [hunk 6]
```diff
@@ -46,279 +45,491 @@ import SendIcon from '@material-ui/icons/Send';
...
```
Comment (raudelf) [inline src/components/navbar.js:455]: commented out code

File: src/components/authentication/Login.js (hunk 1 above)
Comment (PrinceD96) [reply src/components/authentication/Login.js:79]: Yes, Login is still half baked.

File: src/components/navbar.js (hunk 2 above)
Comment (PrinceD96) [reply src/components/navbar.js:152]: Agreed! I haven't looked into this component yet. It was VS code auto-updating the imports.

**Pull request merged**